modules/ossm-observability-visual.adoc

```diff
@@ -5,34 +5,34 @@ This module is included in the following assemblies:
 ////
 
 [id="ossm-observability-visual_{context}"]
-= Working with data in the Kiali console
+= Viewing service mesh data in the Kiali console
 
-From the *Graph* menu in the Kiali console, you can use the following graphs and viewing tools to gain deeper insights about data that travels through your service mesh. These tools can help you identify problems with services or workloads.
+The Kiali Graph offers a powerful visualization of your mesh traffic. The topology combines real-time request traffic with your Istio configuration information to present immediate insight into the behavior of your service mesh, letting you quickly pinpoint issues. Multiple Graph Types let you visualize traffic as a high-level service topology, a low-level workload topology, or as an application-level topology.
 
 There are several graphs to choose from:
 
 * The *App graph* shows an aggregate workload for all applications that are labeled the same.
 
+* The *Service graph* shows a node for each service in your mesh but excludes all applications and workloads from the graph. It provides a high level view and aggregates all traffic for defined services.
+
 * The *Versioned App graph* shows a node for each version of an application. All versions of an application are grouped together.
 
 * The *Workload graph* shows a node for each workload in your service mesh. This graph does not require you to use the application and version labels. If your application does not use version labels, use this the graph.
 
-* The *Service graph* shows a node for each service in your mesh but excludes all applications and workloads from the graph. It provides a high level view and aggregates all traffic for defined services.
+Graph nodes are decorated with a variety of information, pointing out various route routing options like virtual services and service entries, as well as special configuration like fault-injection and circuit breakers. It can identify mTLS issues, latency issues, error traffic and more. The Graph is highly configurable, can show traffic animation, and has powerful Find and Hide abilities.
+
+Click the *Legend* button to view information about the shapes, colors, arrows, and badges displayed in the graph.
 
 To view a summary of metrics, select any node or edge in the graph to display its metric details in the summary details panel.
 
 [id="ossm-observability-topology_{context}"]
-== Namespace graphs
+== Changing graph layouts in Kiali
 
-The namespace graph is a map of the services, deployments, and workflows in your namespace and arrows that show how data flows through them.
+The layout for the Kiali graph can render differently, depending on your application architecture and the data to display, for example, the number of graph nodes and their interactions. Because it is not possible to create a single layout that renders nicely for every situation, Kiali offers a choice of several different layouts.
```

Comment:
I think the readability of this sentence is a bit off: 

The layout for the Kiali graph can render differently, depending on your application architecture and the data to display, for example, the number of graph nodes and their interactions.

It might just be me, and it's fine. My thoughts go somewhere like

The layout for the Kiali graph can render differently depending on your application architecture and the data to display. For example, the number of graph nodes and their interactions can determine how the Kiali graph is rendered. 

Or not. Up to you.